src/NUnitTestAdapter/NUnitEventListener.cs

```diff
@@ -156,16 +155,15 @@ public void TestFinished(XmlNode resultNode)
         public void SuiteFinished(XmlNode resultNode)
         {
             var result = resultNode.GetAttribute("result");
-            var label = resultNode.GetAttribute("label");
             var site = resultNode.GetAttribute("site");
-
+            
```

Comment:
Super silly nitpick: new whitespace added to this line.

Reply:
Happens when a new line is inserted and spaces are default :-(